### DISCUSSION TITLE
Fix ETA for indexing errors with suggestions?

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#12663] [Q&A] [answered]
With a fresh release today, we see that one issue holding us back from upgrading seems to be still unfixed.

The issue is here: https://issues.hibernatingrhinos.com/issue/RavenDB-16762

Previous discussion: https://groups.google.com/g/ravendb/c/w0O362Ntmt0 (there was a follow-up discussion with support where Grisha identified the problem to be related to suggestions in the index).

Is there any estimate about when this issue could be tackled? We'd love to upgrade, but we are stuck on 5.1.5 for the last several months.

**Comment by grisha-kotler** [comment, marked as answer]
We'll have an answer for you during the next sprint.

**Reply by ppekrol** [reply]
@krivulcik we managed to reproduce this in a more consistent manner and we were able to nail down the root cause. Currently we are working on a fix. Will keep you posted.

**Reply by ayende** [reply]
For your information, this is fixed: https://github.com/ravendb/ravendb/pull/12717
